internal/db/schema/migrations/oss/postgres/82/01_citext_extension.up.sql

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+  -- https://www.postgresql.org/docs/14/citext.html allows us to make
+  -- case-insensitive uniqueness constraints which is useful to us for
+  -- aliases.
+  create extension citext;
```

Comment:
If we want to add the citext extension the same way we did for [pgcrypto](https://github.com/hashicorp/boundary/blob/main/internal/db/schema/migrations/oss/postgres/0/60_wh_domain_types.up.sql#L6):
```suggestion
  create extension if not exists "citext";
```